tgui/packages/tgui/interfaces/Pandemic.tsx

```diff
@@ -0,0 +1,484 @@
+import { useBackend, useSharedState } from '../backend';
+import { Box, Button, Collapsible, Input, LabeledList, NoticeBox, ProgressBar, Section, Stack, Tabs, Tooltip } from '../components';
+import { Window } from '../layouts';
+
+type PandemicContext = {
+  beaker?: Beaker;
+  blood?: Blood;
+  has_beaker: number;
+  has_blood: number;
+  is_ready: number;
+  resistances?: Resistance[];
+  viruses?: Virus[];
+};
+
+type Beaker = {
+  volume: number;
+  capacity: number;
+};
+
+type Blood = {
+  dna: string;
+  type: string;
+};
+
+type Resistance = {
+  id: string;
+  name: string;
+};
+
+type Virus = {
+  name: string;
+  can_rename: number;
+  is_adv: number;
+  symptoms: Symptom[];
+  resistance: number;
+  stealth: number;
+  stage_speed: number;
+  transmission: number;
+  index: number;
+  agent: string;
+  description: string;
+  spread: string;
+  cure: string;
+};
+
+type VirusInfoProps = {
+  virus: Virus;
+};
+
+type Symptom = {
+  name: string;
+  desc: string;
+  stealth: number;
+  resistance: number;
+  stage_speed: number;
+  transmission: number;
+  level: number;
+  neutered: number;
+  threshold_desc: Threshold[];
+};
+
+type SymptomDisplayProps = {
+  symptoms: Symptom[];
+};
+
+type Threshold = {
+  label: string;
+  descr: string;
+};
+
+type ThresholdDisplayProps = {
+  thresholds: Threshold[];
+};
+
+export const Pandemic = (_, context) => {
+  const { data } = useBackend<PandemicContext>(context);
+  const { has_beaker, has_blood } = data;
+
+  return (
+    <Window width={650} height={500}>
+      <Window.Content>
+        <Stack fill vertical>
+          <Stack.Item>
+            <BeakerDisplay />
+          </Stack.Item>
+          {!!has_beaker && !!has_blood && (
+            <Stack.Item grow>
+              <SpecimenDisplay />
+            </Stack.Item>
+          )}
+        </Stack>
+      </Window.Content>
+    </Window>
+  );
+};
+
+/** Displays loaded container info, if it exists */
+const BeakerDisplay = (_, context) => {
+  const { act, data } = useBackend<PandemicContext>(context);
+  const { has_beaker, beaker, has_blood } = data;
+  const cant_empty = !has_beaker || !beaker?.volume;
+
+  return (
+    <Section
+      title="Beaker"
+      buttons={
+        <>
```

Comment:
Do you still need the `<>`  and `</>` fragments if they're already within a `<Section>`?

Reply:
Yes because they are within the buttons={} prop
it's a little goofy